docs/proposals/20220411-cluster-api-state-metrics.md

```diff
@@ -10,7 +10,7 @@ reviewers:
   - "@apricote"
   - "@fabriziopandini"
 creation-date: 2022-03-03
-last-updated: 2022-03-15
+last-updated: 2022-08-25
 status: experimental
 ---
```

Comment:
Should we maybe do this in a separate PR? I think there is more outdated in this proposal and I don't really want to block this PR because of that

Reply:
I'd be ok with that, but unsure when merging this then first and updating the proposal afterwards 🤔?!

Reply:
I think it's fine to update the proposal in parallel/separate/afterwards. It wouldn't be the first time we update proposals after we changed the implementation.

We also had clear consensus on the issue that we want to implement it directly in kube-state-metrics instead. So I think we don't need the proposal update PR as some kind of approval.

Reply:
Removed the changes for this file from the PR. I will create a separate one for it 👍